src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -515,6 +516,32 @@ record R(R X)
             Assert.Equal("R..ctor(R X)", initializer.ContainingSymbol.ToTestDisplayString());
         }
 
+        [Fact]
+        public void GetDeclaredSymbolOnFieldInitializer()
```

Comment:
>Field [](start = 39, length = 5)

Field or Property?

Reply:
And the test doesn't do either.

---
In reply to: [605126785](https://github.com/dotnet/roslyn/pull/52256#discussion_r605126785) [](ancestors = 605126785)

Reply:
Changed test name to "Property".
This test is calling getting a declared symbol (out var) that's part of the intializer.

---
In reply to: [605127667](https://github.com/dotnet/roslyn/pull/52256#discussion_r605127667) [](ancestors = 605127667,605126785)